src/com/cft/contactmerge/tests/CompareContactPartsTest.java

```diff
@@ -126,7 +126,7 @@ void doNamesMatch_YesMaybeNo_FirstNamesContainMultiHyphens_CaseInsensitive_Ignor
     }
 
     @Test
-    void doNamesMatch_No_FirstNamesContainMultiHyphens_CaseInsensitive_IgnoreSpaces_DifferentLastName() {
+    void doNamesMatch_No_FirstNamesContainMultiHyphens_DifferentLastName() {
```

Comment:
What do these 14 tests add? We already have a test up above that verifies if the last name is not the same then the result is no. We also have multiple tests showing that if the last name and the first name are equivalent (with and without hyphens) then we get a yes. If they do add something, the test/method name should indicate what is being tested.